schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE patients(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	name VARCHAR(255),
+	data_of_birth DATE,
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	admitted_at DATE,
+	patient_id INT,
+	status VARCHAR(255),
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	type VARCHAR(255),
+	name VARCHAR(255),
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE invoice_items(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	unit_price DECIMAL,
+	quantity INT,
+	total_price DECIMAL,
+	invoices_id INT,
+	treatment_id int,
+	PRIMARY KEY(id)
+);
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 , However  All "join table" for the many-to-many relationships should be created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)